man/bats.1

```diff
@@ -1,115 +1,94 @@
-.\" generated with Ronn/v0.7.3
-.\" http://github.com/rtomayko/ronn/tree/0.7.3
-.
-.TH "BATS" "1" "May 2019" "bats-core" "Bash Automated Testing System"
-.
+.\" generated with Ronn-NG/v0.9.0
+.\" http://github.com/apjanke/ronn-ng/tree/0.9.0
+.TH "BATS" "1" "April 2020" "bats-core" "Bash Automated Testing System"
 .SH "NAME"
 \fBbats\fR \- Bash Automated Testing System
-.
 .SH "SYNOPSIS"
-bats [\-cr] [\-f \fIregex\fR] [\-p | \-t] \fItest\fR\.\.\.
-.
-.br
-bats [\-h | \-v]
-.
+Usage: bats [OPTIONS] \fItests\fR bats [\-h | \-v]
 .P
-\fItest\fR is the path to a Bats test file, or the path to a directory containing Bats test files (ending with "\.bats")\.
-.
+\fItests\fR is the path to a Bats test file, or the path to a directory containing Bats test files (ending with "\.bats")
 .SH "DESCRIPTION"
 Bats is a TAP\-compliant testing framework for Bash\. It provides a simple way to verify that the UNIX programs you write behave as expected\.
-.
 .P
 A Bats test file is a Bash script with special syntax for defining test cases\. Under the hood, each test case is just a function with a description\.
-.
 .P
 Test cases consist of standard shell commands\. Bats makes use of Bash\'s \fBerrexit\fR (\fBset \-e\fR) option when running test cases\. If every command in the test case exits with a \fB0\fR status code (success), the test passes\. In this way, each line is an assertion of truth\.
-.
 .P
 See \fBbats\fR(7) for more information on writing Bats tests\.
-.
 .SH "RUNNING TESTS"
 To run your tests, invoke the \fBbats\fR interpreter with a path to a test file\. The file\'s test cases are run sequentially and in isolation\. If all the test cases pass, \fBbats\fR exits with a \fB0\fR status code\. If there are any failures, \fBbats\fR exits with a \fB1\fR status code\.
-.
 .P
 You can invoke the \fBbats\fR interpreter with multiple test file arguments, or with a path to a directory containing multiple \fB\.bats\fR files\. Bats will run each test file individually and aggregate the results\. If any test case fails, \fBbats\fR exits with a \fB1\fR status code\.
-.
 .SH "OPTIONS"
-.
 .TP
 \fB\-c\fR, \fB\-\-count\fR
 Count the number of test cases without running any tests
-.
 .TP
-\fB\-f\fR, \fB\-\-filter\fR
+\fB\-f\fR, \fB\-\-filter <regex>\fR
 Filter test cases by names matching the regular expression
-.
+.TP
+\fB\-F\fR, \fB\-\-formatter <type>\fR
+Switch between formatters: pretty (default), tap (default w/o term), junit
 .TP
 \fB\-h\fR, \fB\-\-help\fR
-Display help message
-.
+Display this help message
+.TP
+\fB\-j\fR, \fB\-\-jobs <jobs>\fR
+Display this help message
```

Comment:
This is the wrong help text for this option.

Reply:
Well spotted, thank you!